geodata/search.py

```diff
@@ -123,27 +123,21 @@ def search_near_point_geometry(request, *args, **kwargs):
     return JsonResponse(serializers.GeometryEntrySerializer(results, many=True).data, safe=False)
 
 
-def extant_summary(found):
+def extent_summary_spatial(found):
     """
-    Given a query set of items, return a result dictionary with the summary.
+    Given a query set of SpatialEntry, return a result dictionary with the summary.
 
     :param found: a query set with SpatialEntry results.
     :returns: a dictionary with count, collect, convex_hull, extent,
-        acquisition, acqusition_date, created, modified.  collect and
-        convex_hull are geojson objects.
+        acquisition, acqusition_date.  collect and convex_hull are geojson
+        objects.
     """
     if found and found.count():
         summary = found.aggregate(
             Collect('footprint'),
             Extent('footprint'),
             Min('acquisition_date'),
             Max('acquisition_date'),
-            Min('created'),
-            Max('created'),
-            Min('modified'),
-            Max('modified'),
-            acquisition__min=Min(Coalesce('acquisition_date', 'created')),
-            acquisition__max=Max(Coalesce('acquisition_date', 'created')),
```

Comment:
@manthey, I separated this extents summary function to handle `ModifiableEntry` and `SpatialEntry` separately. A consequence is loosing this coalesced field. We could do this manually in `extent_summary` but I don't see an immediate need for it

Reply:
In the endpoints, if we ask to query by acquisition time, and no `acquisition_date` is set, it falls back to using `created`.  The point of this coalesce was to report the min/max of this fallback field.  Can this be moved to the ` extent_summary_modifiable` function?

Reply:
Ah I see, how about inside the new `extent_sumary` function which merges the results of `extent_summary_modifiable` and `extent_summary_spatial`? That's the only place where we can guarantee that both `aquistion_date` and `created` are present

Reply:
Yes, that would work.